.evg.yml

```diff
@@ -90,7 +90,7 @@ tasks:
           aws_key: ${sdks_aws_key}
           aws_secret: ${sdks_aws_secret}
           local_file: stitch-js-sdk/dist/web/stitch.js
-          remote_file: js/library/${revision}/stitch.js
+          remote_file: js/library/${version_id}/stitch.js
```

Comment:
Yes - kept as build artifacts, and then also copied into `stable` when we want to publish a new release.